ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -251,8 +241,7 @@ public void ping(int id) {
     }
     try {
       Buffer buffer = new Buffer();
-      client.sendPing(buffer);
```

Comment:
The public API has been removed, 3.5.0 does automatic pings.
https://github.com/square/okhttp/issues/2902#issuecomment-253106088